rfcs/0000-fix-load-cell-data-hash/0000-fix-load-cell-data-hash.md

```diff
@@ -0,0 +1,37 @@
+---
+Number: "0000"
+Category: Network Protocol (Soft Fork)
+Status: Draft
+Author: Ian Yang
+Organization: Nervos Foundation
+Created: 2021-04-25
+---
+
+# Allow Syscall `load_cell_data_hash` on cell in the tx pool
+
+## Abstract
+
+When the syscalls `load_cell_data` and `load_cell_data_hash` load cell from a transaction still in the pool, they should load correct data and hash as the transaction is already in the chain.
```

Comment:
There are no `load_cell_data_hash` syscall in current version, it should be `load_cell_by_field` with the `field` value 1:
https://github.com/nervosnetwork/rfcs/blob/f0bf9fd6c627f5669e471543d3ba5d62667b592e/rfcs/0009-vm-syscalls/0009-vm-syscalls.md#load-cell-by-field